lib/express-rate-limit.js

```diff
@@ -85,6 +86,18 @@ function RateLimit(options) {
               );
             }
           }
+          if (options.draft_polli_ratelimit_headers) {
+            res.setHeader("RateLimit-Limit", max);
+            res.setHeader("RateLimit-Remaining", req.rateLimit.remaining);
+            if (resetTime instanceof Date) {
+              // if we have a resetTime, also provide the current date to help avoid issues with incorrect clocks
+              res.setHeader("Date", new Date().toGMTString());
```

Comment:
I am not sure we have to provide Date, as it may be set by the application instead.

Date (and other timestamp headers) should be used only in conjunction with NTP or similar mechanism.

Reply:
Yea, we don't need the Date header here. The old "X-RateLimit-Reset" was set to a timestamp, so the date was provided so that endpoints could calculate the offset even if the server or client (or both) had an incorrectly set clock.

With the new "RateLimit-Reset" header, we don't need that because it's already an offset.

Reply:
@nfriedly thanks for all the support you're providing for this PR! 

An advice: the requirement for a middleware to add a  `Date`  header are [here](https://tools.ietf.org/html/rfc7231#section-7.1.1.2)

```
   A recipient *with a clock* that receives a response message without a
   Date header field MUST *record the time it was received* and append a
   corresponding Date header field to the message's header section if it
   is cached or forwarded downstream.
```

Reply:
That makes it sound like the server should handle the date header, not the middleware.

And, some quick testing shows that the node.js http module includes a date header by default, so I should probably remove the existing code that sets the date header also...

Reply:
I'll remove from this then.